CoEpi/domain/logic/CenMatcher.swift

```diff
@@ -56,26 +56,39 @@ class CenMatcherImpl: CenMatcher {
     private func checkForPotentialInfection(cen: CEN, infectedKeys: [CENKey], completion: () -> Void){
         let mod = cen.timestamp % Int64(CenLogic.CENLifetimeInSeconds)
         let roundedLocalTimestamp = cen.timestamp - mod
+        let previousRoundedLocalTimestamp = roundedLocalTimestamp - CenLogic.CENLifetimeInSeconds
         os_log("Local CEN: cen = [ %{public}@ ], timestamp = [ %lld ], rounded timestamp = [ %lld ]", cen.CEN, cen.timestamp, roundedLocalTimestamp)
+        let numOfKeys = infectedKeys.count
         var i : Int = 0
         for key in infectedKeys {
             i+=1
-            let candidateCen = cenLogic.generateCen(CENKey: key.cenKey, timestamp: roundedLocalTimestamp)
-            let candidateCenHex = candidateCen.toHex()
-            os_log("%p %d. candidateCenHex: [%{public}@] based on key [%{public}@ %lld] \n", Thread.current, i, candidateCenHex, key.cenKey, key.timestamp)
-            if cen.CEN == candidateCenHex {
-                os_log("Match found for [ %{public}@ ]", candidateCenHex)
-                //Update matchedKeys on Main thread (preventing race conditions)
-                DispatchQueue.main.async{
-                    self.matchedKeys.append(key)
-                }
+            os_log("Key [ %d/%d ]", i, numOfKeys)
```

Comment:
Can we remove all these logs (including the ones in `matchKeyForTimestamp`)? This is too noisy for the repo. You can leave them commented, if you want.

Reply:
@i-schuetz Its a quick change. 
The idea with the abundant logs was for the advanced testers to be able to track the matching progress using the Console.app